### PR TITLE
Add to delegate job spec CaptureEATelemetry override from evm ocr cfg

### PR DESCRIPTION
**Pull request by ilija42**
This should fix node not sending EnhancedEA telemetry to OTI. CaptureEATelemetry is set in OCR config and not in job spec, so it makes sense to override the value here same as in OCR2 delegate ServicesForSpec.

Added logs for when Ea Telemetry is turned off. Logging is done outside of the func that checks if telemtry is enabled so that the caller has control of logger args

### CODE REVIEW

File: core/services/ocr/delegate.go [hunk 1]
```diff
@@ -295,6 +295,7 @@ func (d *Delegate) ServicesForSpec(jb job.Job) (services []job.ServiceCtx, err e
 			configOverrider = configOverriderService
 		}
 
+		jb.OCROracleSpec.CaptureEATelemetry = chain.Config().OCR().CaptureEATelemetry()
 		enhancedTelemChan := make(chan ocrcommon.EnhancedTelemetryData, 100)
 		if ocrcommon.ShouldCollectEnhancedTelemetry(&jb) {
```
Comment (patrickhuie19) [inline core/services/ocr/delegate.go:300]: Should we add a log line when enhanced telemetry is turned off?

**Review by patrickhuie19 — APPROVED** [review]

File: core/services/ocr/delegate.go (hunk 1 above)
Comment (ilija42) [reply core/services/ocr/delegate.go:300]: added

File: core/services/ocr/delegate.go [hunk 2]
```diff
@@ -295,10 +295,13 @@ func (d *Delegate) ServicesForSpec(jb job.Job) (services []job.ServiceCtx, err e
...
 			enhancedTelemService := ocrcommon.NewEnhancedTelemetryService(&jb, enhancedTelemChan, make(chan struct{}), d.monitoringEndpointGen.GenMonitoringEndpoint("EVM", chain.ID().String(), concreteSpec.ContractAddress.String(), synchronization.EnhancedEA), lggr.Named("EnhancedTelemetry"))
 			services = append(services, enhancedTelemService)
+		} else {
+			lggr.Infow("Ee telemetry is disabled for job", "job", jb.Name)
```
Comment (jmank88) [inline core/services/ocr/delegate.go:304]: Enhanced?
```suggestion
			lggr.Infow("Enhanced telemetry is disabled for job", "job", jb.Name)
```

File: core/services/ocr2/delegate.go [hunk 3]
```diff
@@ -746,6 +746,8 @@ func (d *Delegate) newServicesMedian(
 	if ocrcommon.ShouldCollectEnhancedTelemetry(&jb) {
 		enhancedTelemService := ocrcommon.NewEnhancedTelemetryService(&jb, enhancedTelemChan, make(chan struct{}), d.monitoringEndpointGen.GenMonitoringEndpoint(rid.Network, rid.ChainID, spec.ContractID, synchronization.EnhancedEA), lggr.Named("EnhancedTelemetry"))
 		medianServices = append(medianServices, enhancedTelemService)
+	} else {
+		lggr.Infow("Ee telemetry is disabled for job", "job", jb.Name)
```
Comment (patrickhuie19) [inline core/services/ocr2/delegate.go:750]:
```suggestion
		lggr.Infow("Enhanced telemetry is disabled for job", "job", jb.Name)
```

File: core/services/ocrcommon/data_source.go [hunk 4]
```diff
@@ -144,6 +144,8 @@ func (ds *inMemoryDataSource) executeRun(ctx context.Context, timestamp Observat
 			FinalResults:   finalResult,
 			RepTimestamp:   timestamp,
 		})
+	} else {
+		ds.lggr.Infow("Ee telemetry is disabled for job", "job", ds.jb.Name)
```
Comment (patrickhuie19) [inline core/services/ocrcommon/data_source.go:148]:
```suggestion
		ds.lggr.Infow("Enhanced telemetry is disabled for job", "job", ds.jb.Name)
```

**Review by gheorghestrimtu — APPROVED** [review]

**Pull request merged**